libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Schemas/Actions/Teams.SendTabAuthResponse.schema

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IDialog)",
+    "title": "Send tab auth response",
+    "description": "If the user is signed in, the dialog will end setting the TokenResponse in the specified property.",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "title": "Id",
+            "description": "Optional id for the dialog"
+        },
+        "connectionName": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "OAuth connection name",
+            "description": "The OAuth Connection Name, that will be used to perform the Sign On.",
```

Comment:
>Sign On [](start = 88, length = 7)

Why capitalized?

Reply:
I favor the lower ascii ranges. ?

#Fixed